configure.py.patch

```diff
@@ -0,0 +1,95 @@
+--- a/lib/configure.py	2023-01-01 18:36:03.000000000 -0500
++++ b/lib/configure.py	2023-11-27 09:29:24.308414038 -0500
+@@ -477,7 +477,7 @@
+     ''' The same as checkProgAlternatives, but for viewers '''
+     alt_rc_entry = listAlternatives(progs, 'viewer', rc_entry)
+     return checkProgAlternatives(description, progs, rc_entry,
+-                                 alt_rc_entry, path, not_found = 'auto')
++                                 alt_rc_entry, path, not_found = 'xdg-open')
+ 
+ 
+ def checkEditor(description, progs, rc_entry = [], path = []):
+@@ -1888,7 +1888,77 @@
+         removeFiles(['chkconfig.vars', 'chklatex.ltx', 'chklatex.log',
+             'chklayouts.tex', 'chkmodules.tex', 'chkciteengines.tex',
+             'missfont.log', 'wrap_chkconfig.ltx', 'wrap_chkconfig.log'])
++            
++            
++def additional_settings():
++	
++	__Aname="/.tmplyx"
++	
++	__Aoutput = subprocess.run(['xdg-user-dir', 'DOCUMENTS'], capture_output=True, text=True)
```

Comment:
Do I understand your approach correctly that you write the temporary files into `XDG_DOCUMENTS_DIR/.tmplyx/lyx_tmpdir.fRPnRZiOjAbE` (the name of the last subfolder is arbitrary)?
Is there a clean up routine? In my case, I have two folders left from previous lyx (flatpak) starts.

Reply:
The folder names are arbitrary.
As far as I had tried, the folder cleaned itself, but now I see that I have one folder left, I don't know in what context it is created. Is your directory filled with empty folders?
I don't know if this is the best scenario to discuss these topics. If you prefer, you can write to me at sixpalax@gmail.com, the discussion can be extensive.